tests/Microsoft.Identity.Test.Unit/BrokerTests/RuntimeBrokerTests.cs

```diff
@@ -71,7 +71,7 @@ public void WamOnWin10()
                .Create("d3adb33f-c0de-ed0c-c0de-deadb33fc0d3")
                .WithAuthority(TestConstants.AuthorityTenant);
 
-            pcaBuilder = pcaBuilder.WithBrokerPreview();
+            pcaBuilder = pcaBuilder.WithWindowsBroker();
```

Comment:
Why is the test failing?

Reply:
It fails with error unable to load msal_runtime.